api/v1alpha1/combination_types.go

```diff
@@ -27,6 +27,8 @@ const (
 	ReasonTemplateBodyInvalid = "TemplateBodyInvalid"
 	ReasonEvaluationsInvalid  = "EvaluationsInvalid"
 	ReasonProcessed           = "Processed"
+	ReasonCreationFailed      = "CreationFailed"
```

Comment:
nit: I'd say `ApplyFailed`/`ApplySucceeded` makes this a bit clearer since the flag is `Apply`. What are your thoughts on that?

Reply:
that makes sense 👍